WorkloadManagementSystem/Agent/SiteDirector.py

```diff
@@ -446,6 +448,24 @@ def submitJobs( self ):
       platform = self.queueDict[queue]['Platform']
       siteMask = siteName in siteMaskList
 
+      # Check the status of the Site
+      result = self.sstClient.getSiteStatuses({siteName})
+      if result['Value']:
```

Comment:
If the site does not exists in the list given to `getSiteStatuses` an empty value will be returned and therefore an error will be triggered in the next line because the `result['Value'][siteName]` will be non existent.

For that reason, I decided to check the result['Value']